db/database.go

```diff
@@ -666,6 +667,14 @@ func (context *DatabaseContext) stopBackgroundManagers() []*BackgroundManager {
 		}
 	}
 
+	if context.AsyncIndexInitManager != nil {
```

Comment:
I'd be tempted to just construct just to `for _, mgr := range []*BackgroundManager{context.ResyncManager, context.AttachmentCompactionManager...}` to dedupe this code a bit.